web/html/meta-elements/author.md

```diff
@@ -37,7 +37,10 @@ The author metadata has no effect on search engine optimization or SEO.
 
 Write the meta tag that states that 'John Doe' built the web page.
 
-<??? ????="???" ????="???">
+```
```

Comment:
Should we add the `html` tag here?

\`\`\`html

\`\`\`

Reply:
probably we'll do this with a script to the whole HTML subtopic